bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -33,11 +33,11 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
+    operators.operatorframework.io/builder: operator-sdk-v1.13.1
```

Comment:
The `operator-sdk` version we are using in CI is `operator-sdk-v1.13.0+git`, Can you switch the version in your local system to thedesired one? Also, the operator-sdk version semantics differ from the source installed. Below is the discussion on the same we had previously:  

https://github.com/open-telemetry/opentelemetry-operator/pull/336#discussion_r665022289

This version change is causing CI to fail.

Reply:
Fixed.

Reply:
Fixed.